cmd/offline/offline.go

```diff
@@ -70,6 +63,29 @@ func SaveHeartbeats(v *viper.Viper, heartbeats []heartbeat.Heartbeat) error {
 	return nil
 }
 
+func loadParams(v *viper.Viper) (paramscmd.Params, error) {
+	paramHeartbeat, err := paramscmd.LoadHeartbeatParams(v)
+	if err != nil {
+		return paramscmd.Params{}, fmt.Errorf("failed to load heartbeat parameters: %s", err)
+	}
+
+	paramAPI, err := paramscmd.LoadAPIParams(v)
```

Comment:
We don't need to load api params when saving heartbeats to the offline db.

Reply:
It's required for `setLogFields()` [here](https://github.com/wakatime/wakatime-cli/blob/misc/improve-load-params/cmd/offline/offline.go#L141) and `buildHeartbeats()` [here](https://github.com/wakatime/wakatime-cli/blob/misc/improve-load-params/cmd/offline/offline.go#L93).

Reply:
But we don't want the param heartbeat when only saving surplus extra heartbeats here:
https://github.com/wakatime/wakatime-cli/blob/f008c674afa6615bbaacacfb0c50d2401da74eb5/cmd/heartbeat/heartbeat.go#L97

Reply:
Oh, f008c674afa6615bbaacacfb0c50d2401da74eb5 fixes it 👍